daffodil-runtime1/src/main/scala/org/apache/daffodil/debugger/InteractiveDebugger.scala

```diff
@@ -1711,6 +1711,7 @@ class InteractiveDebugger(runner: InteractiveDebuggerRunner, eCompilers: Express
             case VariableRead => "read"
             case VariableSet => "set"
             case VariableUndefined => "undefined"
+            case VariableBeingDefined => "being defined"
```

Comment:
How is VariableBeingDefined different from VariableInProcess? My reading of the scaladoc for VariableInProcess sounds alot like what we need. When a NVI is suspended, what if we just put that instance into VariableInProcess? This says that any read should suspend. And If we implement Mike's suggestion, set's aren't even allowed if there is a default value. So basically VariableInProcess means this variable is being set by either a defaultValue or setVariable (but only via one of them) and that suspension hasn't finished yet. Once that's finished, reads can unsuspend.